proc_limits.go

```diff
@@ -103,8 +103,7 @@ func (p Proc) Limits() (ProcLimits, error) {
 		//fields := limitsMatch.Split(s.Text(), limitsFields)
 		fields := limitsMatch.FindStringSubmatch(s.Text())
 		if len(fields) != limitsFields {
-			return ProcLimits{}, fmt.Errorf(
-				"couldn't parse %s line %s", f.Name(), s.Text())
+			return ProcLimits{}, fmt.Errorf("couldn't parse %s line %s", f.Name(), s.Text())
```

Comment:
Perhaps `"couldn't parse %s line: %q"` would be more consistent? Depending on the nature of `f.Name()`, it might deserve a %q, too.